enterprise/frontend/src/metabase-enterprise/audit_app/components/AuditNotificationDeleteModal/AuditNotificationDeleteModal.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import { t } from "ttag";
+import { formatChannels } from "metabase/lib/notifications";
+import Button from "metabase/components/Button";
+import CheckBox from "metabase/components/CheckBox";
+import FormMessage from "metabase/components/form/FormMessage";
+import ModalContent from "metabase/components/ModalContent";
+import { CheckboxLabel } from "./AuditNotificationDeleteModal.styled";
+
+const propTypes = {
+  item: PropTypes.object.isRequired,
+  type: PropTypes.oneOf(["alert", "pulse"]).isRequired,
+  onDelete: PropTypes.func,
+  onClose: PropTypes.func,
+};
+
+const AuditNotificationDeleteModal = ({ item, type, onDelete, onClose }) => {
+  const [error, setError] = useState();
+  const [checked, setChecked] = useState(false);
+
+  const handleDeleteClick = async () => {
+    try {
+      await onDelete(item, true);
+      onClose(true);
+    } catch (error) {
+      setError(error);
+    }
+  };
+
+  const handleCheckedChange = event => {
+    setChecked(event.target.checked);
+  };
+
+  const handleClose = () => onClose(true);
+
+  return (
+    <ModalContent
+      title={getTitleMessage(item, type)}
+      footer={[
```

Comment:
Could the contents of this footer prop be:

1. in a React Fragment?
2. set outside ModalContent, so that the setting of prop looks kind of like `footer={footerContent}`?

🤔

Reply:
I have to keep it being an array because of the ModalContent implementation that adds paddings